mesheryctl/internal/cli/root/system/system.go

```diff
@@ -37,10 +37,12 @@ var SystemCmd = &cobra.Command{
 	Use:   "system",
 	Short: "Meshery Lifecycle Management",
 	Long:  `Manage the state and configuration of Meshery server, components, and client.`,
-	Args:  cobra.MinimumNArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
```

Comment:
Similarly in other commands as well.

```suggestion
		if len(args) == 0 {
```

It can give incorrect error messages in some cases. For example, here if a user slightly misspells `context`, this gives the following output.

```
➜ mesheryctl system contexta view
Error: mesheryctl system requires at least 1 arg(s), got only 0

See https://docs.meshery.io/reference/mesheryctl/system for usage details
```

Reply:
Thanks @nishantwrp for pointing that out.